### PR TITLE
BUG: Upgrade to JDK11

### LINKED ISSUE #1936: BUG: CI is failing due the new continuumio/miniconda3:latest
```sh
Status: Downloaded newer image for continuumio/miniconda3:latest
 ---> 83375d57598f
Step 2/10 : RUN apt-get -qq update --yes  && apt-get -qq install --yes --no-install-recommends     build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre  && rm -rf /var/lib/apt/lists/*
 ---> Running in 7fc6867a735d
E: Unable to locate package openjdk-8-jre‌
Service &#x27;ibis&#x27; failed to build: The command &#x27;&#x2F;bin&#x2F;sh -c apt-get -qq update --yes  &amp;&amp; apt-get -qq install --yes --no-install-recommends     build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre  &amp;&amp; rm -rf &#x2F;var&#x2F;lib&#x2F;apt&#x2F;lists&#x2F;*&#x27; returned a non-zero code: 100‌
##[error]Bash exited with code '1'.
```

### PR DESCRIPTION
**Pull request by cpcloud**
Closes #1936

### CODE REVIEW

**Comment by xmnlab** [comment]
it seems it has some conflict with spark

File: ci/Dockerfile.dev [hunk 1]
```diff
@@ -2,19 +2,19 @@ FROM continuumio/miniconda3
 
 RUN apt-get -qq update --yes \
  && apt-get -qq install --yes --no-install-recommends \
-    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-8-jre \
+    build-essential git make clang libboost-dev postgresql-client ca-certificates openjdk-11-jre \
```
Comment (xmnlab) [inline ci/Dockerfile.dev]: maybe you could just remove openjdk here and install that using conda

File: ci/Dockerfile.dev [hunk 2]
```diff
@@ -2,19 +2,19 @@ FROM continuumio/miniconda3
...
 
-RUN conda config --add channels conda-forge \
-  && conda update --all --yes --quiet \
-  && conda env create --name ibis-env --file /requirements-$PYTHON-dev.yml \
-  && conda install --yes conda-build \
-  && conda clean --all --yes
+RUN /opt/conda/bin/conda config --add channels conda-forge \
+  && /opt/conda/bin/conda update --all --yes --quiet \
+  && /opt/conda/bin/conda env create --name ibis-env --file /requirements-$PYTHON-dev.yml \
+  && /opt/conda/bin/conda install --yes conda-build \
```
Comment (xmnlab) [inline ci/Dockerfile.dev]: and maybe you can add something like:

`  && /opt/conda/bin/conda install -n ibis-env --yes openjdk=8 \`

Comment (cpcloud) [reply ci/Dockerfile.dev]: I placed the specification of the JDK in dev requirements files.

**Pull request merged**